env/build_singularity_gnu.env

```diff
@@ -0,0 +1,17 @@
+#Setup instructions for singularity container using gnu 9.3.0 built from this docker image: docker://noaaepic/ubuntu20.04-epic-srwapp:1.0
+#https://hub.docker.com/r/noaaepic/ubuntu20.04-epic-srwapp
+
+source /usr/share/lmod/6.6/init/profile
+
+module purge
+
+module use /opt/hpc-modules/modulefiles/stack
+
+module load hpc hpc-gnu hpc-openmpi
```

Comment:
It is often easier to manage lists of modules when they are one-per-line. Would you mind making that change?